### DISCUSSION TITLE
Hyped auf Styx 4.4.0

### DISCUSSION THREAD

**Discussion by Pflegecraft** [ophian/styx#52] [General]
Bin schon sehr Hyped auf Styx 4.4.0 weiß man schon wann der Release ansteht?
Achso sehr geile Arbeit übrigends und auch das du die Plugins so Aktuell hälst :)

Lieben Gruß
Pflegecraft

**Comment by ophian** [comment]
Die Beta, ... bald würde ich sagen.
Danke. Übrigens: Bei Hype(s) hilft: 3 x tief durchatmen.
